cmd/fluentd/lib/fluent/plugin/out_loki.rb

```diff
@@ -155,7 +158,7 @@ def ssl_opts(uri)
 
         if !@cert.nil? && !@key.nil?
           opts = opts.merge(
-            verify_mode: OpenSSL::SSL::VERIFY_PEER,
+            verify_mode: @verify_tls ? OpenSSL::SSL::VERIFY_PEER : OpenSSL::SSL::VERIFY_NONE,
```

Comment:
This code path is about not using a client certificate, the common case.

I think that even when using a client certificate one might want to be able to disable server certificate verification which makes me think that `@verify_tls` should be used in some other code path, too. WDYT?

Reply:
That is, further above we might want to have something in the lines of
```
        opts = {
          use_ssl: uri.scheme == 'https',
          verify_mode: @verify_tls ? OpenSSL::SSL::VERIFY_PEER : OpenSSL::SSL::VERIFY_NONE,
        }
```

Reply:
According to [rdoc](https://ruby-doc.org/stdlib-2.7.1/libdoc/openssl/rdoc/OpenSSL/SSL/SSLContext.html), the default value is `VERIFY_NONE`. 


>Valid modes are VERIFY_NONE, VERIFY_PEER, VERIFY_CLIENT_ONCE, VERIFY_FAIL_IF_NO_PEER_CERT and defined on OpenSSL::SSL, The default mode is VERIFY_NONE, which does not perform any verification at all.


I keep the current default `VERIFY_PEER` but enable developers to turn off the verification with `verify_tls = false` https://github.com/grafana/loki/pull/2171/commits/07e0defa5b19ad894dba840d5402a598c507264e

WDYT?